commit e13b0e83911bad8b8915d92ce8f5622d9707cbb7 (refs/changes/79/2379/2)
Author: Kody A Kantor <kody.kantor@gmail.com>
Date:   2017-08-11T18:03:20+00:00 (2 years, 2 months ago)
    
    MANATEE-350 Backup process fails due to pg_dump not being in PATH

diff --git a/pg_dump/pg_backup_common.sh b/pg_dump/pg_backup_common.sh
index 6bfd0c9..983babd 100644
--- a/pg_dump/pg_backup_common.sh
+++ b/pg_dump/pg_backup_common.sh
@@ -16,7 +16,7 @@ export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}
 set -o xtrace
 set -o pipefail
 
-PATH=/opt/smartdc/manatee/node_modules/.bin:/opt/smartdc/manatee/build/node/bin:/opt/local/bin:/usr/sbin/:/usr/bin:/usr/sbin:/usr/bin:/opt/smartdc/registrar/build/node/bin:/opt/smartdc/manatee/node_modules/.bin:/opt/smartdc/registrar/node_modules/.bin:/opt/smartdc/manatee/lib/tools:/opt/smartdc/manatee/pg_dump/
+PATH=$PATH:/opt/smartdc/manatee/node_modules/.bin:/opt/smartdc/manatee/pg_dump/
 
 FATAL=
 CFG=/opt/smartdc/manatee/etc/backup.json
